test/integration/buoyancy.cc

```diff
@@ -42,6 +42,121 @@ class BuoyancyTest : public InternalFixture<::testing::Test>
 {
 };
 
+/////////////////////////////////////////////////
+TEST_F(BuoyancyTest, IGN_UTILS_TEST_DISABLED_ON_WIN32(RestoringMoments))
+{
+  // This test checks if the restoring moments are correctly calculated accross
+  // both uniform and graded modes when the vehicle is fully submerged.
+  std::size_t iterations{10000};
+  std::vector<math::Pose3d> poses_uniform, poses_graded;
```

Comment:
```suggestion
  std::vector<math::Pose3d> posesUniform, posesGraded;
```

Style nit. There are more instances throughout the test.